resources/views/column.blade.php

```diff
@@ -5,6 +5,7 @@
     'danger' => 'bg-danger-600',
     'success' => 'bg-success-600',
     'warning' => 'bg-warning-600',
+    default => $getColor()
```

Comment:
Can we store the result of `$getColor()` somewhere so we don't need to invoke the callback twice?

Reply:
sure, should be fixed